src/xdocs/writingchecks.xml

```diff
@@ -176,16 +178,15 @@ CLASS_DEF -> CLASS_DEF [5:0]
       </div>
       <p>
         As you can see very small java file transforms to a huge Abstract Syntax Tree, because
-        that is the most detailed tree including all components of the java file: object block,
+        this detailed tree includes all components of the java file: object blocks,
         comments, modifiers, etc.
       </p>
     </section>
 
     <section name="The Checkstyle SDK Gui">
-
       <p>
         Still with us? Great, you have mastered the basic theory so here
-        is your reward - a GUI that displays the structure of a Java source file. To run it type
+        is your reward - a GUI that displays the structure of a Java source file. To run it, run
```

Comment:
I am not liking the double `run`. Its like we are saying "To run it, you must run it". It doesn't sound professional. I again like the original `type`.

Reply:
I don't think I agree: the goal is not to type it, the goal is to run it - you'll probably copy paste this anyway. Changed to: `Start the GUI with`

Reply:
Good suggestion, done.